packages/shell-api/src/helpers.ts

```diff
@@ -794,3 +794,15 @@ export function getBadge(collections: Document[], index: number): string {
 
   return '';
 }
+
+export const FREE_MONITORING_BANNER = `\
+Enable MongoDB's free cloud-based monitoring service, which will then receive and display
+metrics about your deployment (disk utilization, CPU, operation statistics, etc).
+
+The monitoring data will be available on a MongoDB website with a unique URL accessible to you
+and anyone you share the URL with. MongoDB may use this information to make product
+improvements and to suggest MongoDB products and deployment options to you.
+
+To enable free monitoring, run the following command: db.enableFreeMonitoring()
+To permanently disable this reminder, run the following command: db.disableFreeMonitoring()
```

Comment:
Can't think of anything to improve, but this might be a good time to check with Max or product if we want to update/improve this message https://github.com/mongodb/mongo/blob/2cba27684ff70e38b79bc01d7b570b0af6f4a068/src/mongo/shell/utils.js#L1153-L1167
Nothing to change, something I wanted to mention in case anyone thinks of ways to improve it. (Feel free to resolve).

Reply:
Yeah, pinged Max about this, he said we’ll leave it like this for now and can get more input from product on this later